lib/linguist/languages.yml

```diff
@@ -5473,6 +5473,18 @@ Vim script:
   - vimrc
   ace_mode: text
   language_id: 388
+Vim snippet:
```

Comment:
Could you capitalise `Snippet`, please?

~~~diff yml
-Vim snippet
+Vim Snippet
~~~

Reply:
I didn't capitalise it because I saw that `Vim script` is lowercase.

Reply:
Normally, we prefer to keep things capitalised, unless non-standard capitalisation is canon.

I chose to name `Vim script` with a lowercase `s` because that's apparently how Bram Moolenaar always referred to the language. The entry was previously named `VimL`, which wasn't just incorrect, but the name appeared to have originated on GitHub and began creeping into casual usage.

I wanted to make up for that, so I was cautious with the precise spelling I used.

By no means a common reason for an exception, but there you go. 😉